### ISSUE TITLE
Icinga2 ended ... Failed!

### ISSUE THREAD

**Issue by Maxwell-Mc** [jjethwa/icinga2#218]
my docker logs are :

Starting icinga2 monitoring daemon in foreground.,
[2020-01-29 15:42:54 +0000] information/cli: Icinga application loader (version: r2.10.3-1),
[2020-01-29 15:42:54 +0000] critical/cli: Another instance of Icinga already running with PID 1709,
icinga2 ended ... failed!,
ERROR: Zend_Db_Statement_Exception in /usr/share/icingaweb2/library/vendor/Zend/Db/Statement/Pdo.php:225 with message: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'icingaweb2_director.icinga_apiuser' doesn't exist, query was: SELECT icinga_apiuser.* FROM icinga_apiuser WHERE (object_name = 'icinga2-director'),
2020-01-29 15:42:57,663 INFO exited: icinga2 (exit status 1; not expected),
2020-01-29 15:42:58,669 INFO spawned: 'icinga2' with pid 5088,
checking Icinga2 configuration2020-01-29 15:42:59,706 INFO success: icinga2 entered RUNNING state, process has stayed up for > than 1 seconds (startsecs),

can you help me understand this?

**Comment by jjethwa** [comment]
Hi @Maxwell-Mc 

Is this using latest?  Are you using an external database?

**Comment by Maxwell-Mc** [comment]
I'm using an external mysql with your compose yml and yes latest.

**Comment by jjethwa** [comment]
Thanks for the information.  If this is a brand new setup, make sure you've run through all of the MySQL/MariaDB SQL scripts.  If it's an upgrade, you just need to run the upgrade scripts.

new: https://icinga.com/docs/icinga2/latest/doc/02-installation/#setting-up-the-mysql-database
upgrade: https://icinga.com/docs/icinga2/latest/doc/16-upgrading-icinga-2/#upgrading-the-mysql-database

**Comment by Maxwell-Mc** [comment]
But doesn't your repository contain scripts that import the database schemas? Shouldn't a docker-compose up -d bring it all up?

**Comment by jjethwa** [comment]
Yes, the initial setup should all be taken care of, but upgrades are a bit more complicated

**Comment by StefanWetterActiware** [comment]
Hello Jordan,
do i get that right, that in case of any update i have to apply these scripts manually? Or is it done by your image in some circumstances?
Thanks for your great work!

**Comment by jjethwa** [comment]
Hi @StefanWetterActiware 

That's correct, the database upgrade scripts will need to be run manually when updating to new versions :smile:

**Comment by jjethwa** [comment]
Hi @jhaumanath 

If you are receiving the error when upgrading to newer versions of the container, you to need to run the database upgrade scripts manually.  Unfortunately, there's no way to automate that part
